dev-libs/libsodium/libsodium-1.0.16.ebuild

```diff
@@ -37,7 +38,7 @@ src_configure() {
 		${myconf}
 }
 
-src_install() {
+src_multilib_install() {
 	default
 	prune_libtool_files
```

Comment:
While at it, could you look if all `.la` files could be stripped and replace it with:

    find "${D}" -name '*.la' -delete || die

please?

Reply:
Well, when I do delete all *.la files, I am still able to compile unbound, so this should be ok?